evap/student/templates/student_index.html

```diff
@@ -81,34 +81,44 @@
                                             {% endif %}
                                         {% endif %}
                                     </td>
-                                    <td class="text-right">
-                                        {% if course.grades_activated and course.grade_documents.count == 1 and can_download_grades %}
-                                            <a class="btn btn-sm btn-light" href="{% url 'grades:download_grades' course.grade_documents.all.first.id %}" role="button">{{ course.grade_documents.all.first.description }}</a>
-                                        {% elif course.grade_documents.count > 1 and can_download_grades %}
-                                            <div class="btn-group" role="group">
-                                                <a id="btnDownload{{ course.id }}" role="button" class="btn btn-sm btn-light dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">{% trans 'Grades' %}</a>
-                                                <div class="dropdown-menu" aria-labelledby="btnDownload{{ course.id }}">
+                                    {% if can_download_grades %}
```

Comment:
`and semester|are_grades_activated`? because for these semesters that column will be empty for no apparent reason. the alternative would be to explain the reason somehow.

Reply:
check